tools/self-hosted-runner-utils/clear_offline_runners.py

```diff
@@ -0,0 +1,56 @@
+import os
+import argparse
+import re
+
+from github import Github
+
+def parse_args() -> argparse.Namespace:
+    parser = argparse.ArgumentParser(
+        description='Clear offline self hosted runners for Github repositories'
+    )
+    parser.add_argument(
+        "repo",
+        help="Repository to remove offline self hosted runners for, (ex. pytorch/pytorch)",
+        type=str
+    )
+    parser.add_argument(
+        "--dry-run",
+        help="Don't actually remove the runners, just output which runners would be removed",
+        action='store_true',
+        default=False
+    )
+    parser.add_argument(
+        "--include",
+        help="Regex to match runner names to remove offline runners of",
+        type=str,
+        default=".*"
+    )
+    parser.add_argument(
+        "--token",
+        help="Github token to pull from (Can also pass GITHUB_TOKEN as an env variable)",
+        type=str,
+        default=os.getenv("GITHUB_TOKEN", "")
+    )
+    options = parser.parse_args()
+    return options
+
```

Comment:
Did you run flake8 on this one? I shouldn't there be 2 newlines between functions?

Reply:
If we care about this stuff we should a gha workflow for it

Reply:
Added black formatting as well as a new github actions workflow to check formatting